src/commands/junit/id.ts

```diff
@@ -0,0 +1,64 @@
+/* eslint-disable */
+// Eslint disabled because we're doing bitwise logic, which isn't allowed by the linter
+// From https://github.com/DataDog/dd-trace-js/blob/e4b9a268f0429b6f1e92c384b61a0d104aeb1259/packages/dd-trace/src/id.js
```

Comment:
Would it be worth grabbing a test out of https://github.com/DataDog/dd-trace-js/blob/e4b9a268f0429b6f1e92c384b61a0d104aeb1259/packages/dd-trace/test/id.spec.js , too?

Reply:
yeah that makes sense

Reply:
done in https://github.com/DataDog/datadog-ci/pull/932/commits/bb8edbed0d9dfa42e5e322e39760af382b5a5977